app-modules/service-management/src/Models/ServiceRequestStatus.php

```diff
@@ -63,13 +67,30 @@ class ServiceRequestStatus extends BaseModel implements Auditable
         'name',
         'color',
         'is_system_protected',
+        'sort',
     ];
 
     public function serviceRequests(): HasMany
     {
         return $this->hasMany(ServiceRequest::class, 'status_id');
     }
 
+    public function update(array $attributes = [], array $options = []): bool
```

Comment:
I don't know about modifying this just to throw a generic `Exception`. We already have SQL protection against editing these so the best this is doing is adding some complexity for the reduction of an invalid SQL call which will translate to a named exception anyways.

Unless you feel there is a strong reason for this, I would remove it.